checks/apps/gromacs/gromacs_stmv.py

```diff
@@ -1,122 +1,78 @@
 import os
 import reframe as rfm
 import reframe.utility.sanity as sn
-from hpctestlib.sciapps.gromacs.benchmarks import gromacs_check
-
-# This is based on CSCS Reframe GROMACS tests library.
-# The test uses STMV benchmark (https://doi.org/10.5281/zenodo.3893789),
-# with a different GPU acceleration modes (update: gpu resident and gpu offload; bonded and non-bonded interactions on gpu),
-# evalutes performance of a single node configuration with heFFTe and VkFFT libraries and a two-node with heFFTe only,
-# checks for a total energy on step 0 and conserved energy drift against reference values.    
 
 @rfm.simple_test
-class lumi_gromacs_stmv(gromacs_check):
-    # Purpose of a second and a third parameter changes to a total energy 
-    # at step 0 and energy drift; tolerances are now in readout functions
-    benchmark_info = parameter([
-        ('stmv', [-1.45939e+07, 1.40e-03], [0.001, 0.1]), 
-    ], fmt=lambda x: x[0], loggable=True)
-    update_mode = parameter(['gpu', 'cpu'])
-    nb_impl = parameter(['gpu'])
-    bonded_impl = parameter(['gpu'])
-    fft_variant = parameter(['heffte', 'vkfft'])
+class lumi_gromacs_stmv(rfm.RunOnlyRegressionTest):
+    '''GROMACS STMV benchmark. Updated version.
```

Comment:
@pszi1ard I would appreciate some reference to mention here.

Reply:
Here's the upload of you paper's supplementary info: https://zenodo.org/doi/10.5281/zenodo.11087334

The direct link to the improved STMV input is: https://zenodo.org/records/11087335/files/stmv_gmx_v2.tar.gz?download=1